### PR TITLE
add recipe for okteta 0.26.7

### PR DESCRIPTION
**Pull request by ivanholmes**
Hex editor from KDE

### CODE REVIEW

File: kde-apps/okteta/okteta-0.26.7.recipe [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+* Editing and navigating similar to a text editor
+* Customizable data views, with loadable and storable profiles
+* Tools dockable on all sides or floating
+* Numerical encodings: Hexadecimal, Decimal, Octal, Binary
+* Character encodings: All 8-bit encodings as supplied by Qt, EBCDIC
+* Fast data rendering on screen
+* Multiple open files
+* Undo/redo support
+* Structures tool for analyzing and editing based on user-creatable structure definitions"
+HOMEPAGE="https://apps.kde.org/okteta/https://apps.kde.org/okteta/"
```
Comment (augiedoggie) [inline kde-apps/okteta/okteta-0.26.7.recipe:16]:
```suggestion
HOMEPAGE="https://apps.kde.org/okteta/"
```

**Review by augiedoggie — CHANGES_REQUESTED** [review]

**Comment by augiedoggie** [comment]
Seems OK, I was able to build and run it on 32 bit beta 3.  Would be nice to get a TEST() section added.

**Comment by ivanholmes** [comment]
> Seems OK, I was able to build and run it on 32 bit beta 3. Would be nice to get a TEST() section added.

Agreed. I'll look into this

**Comment by ivanholmes** [comment]
> > Seems OK, I was able to build and run it on 32 bit beta 3. Would be nice to get a TEST() section added.
> 
> Agreed. I'll look into this

Done. Only 41% of them pass and I haven't looked into why. Hopefully not a serious problem

**Review by augiedoggie — APPROVED** [review]
I'll give it a day or so before merging to see if anyone wants to look it over.

**Comment by korli** [comment]
A script struct2osd should be found in $binDir when I check the contents of the arch linux package.

**Comment by ivanholmes** [comment]
> 

> A script struct2osd should be found in $binDir when I check the contents of the arch linux package.

Fixed now along with the locations of other associated files.

File: kde-apps/okteta/okteta-0.26.7.recipe [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$binDir \
+		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
+		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
+		-DCMAKE_INSTALL_LIBDIR=$libDir \
+		-DCMAKE_INSTALL_BINDIR=$binDir \
+		-DSHARE_INSTALL_PREFIX=$dataDir \
```
Comment (Begasus) [inline kde-apps/okteta/okteta-0.26.7.recipe:175]: These could be combined (not tested yet)
```diff
	-DCMAKE_BUILD_TYPE=Release \
-		-DCMAKE_INSTALL_PREFIX=$binDir \
-		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
-		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
-		-DCMAKE_INSTALL_LIBDIR=$libDir \
-		-DCMAKE_INSTALL_BINDIR=$binDir \
+		$cmakeDirArgs \
		-DSHARE_INSTALL_PREFIX=$dataDir \
```

Comment (Begasus) [inline kde-apps/okteta/okteta-0.26.7.recipe:175]: These could be combined (not tested yet)
```diff
	-DCMAKE_BUILD_TYPE=Release \
-		-DCMAKE_INSTALL_PREFIX=$binDir \
-		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
-		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
-		-DCMAKE_INSTALL_LIBDIR=$libDir \
-		-DCMAKE_INSTALL_BINDIR=$binDir \
+		$cmakeDirArgs \
		-DSHARE_INSTALL_PREFIX=$dataDir \
```

**Comment by Begasus** [comment]
Is the nanosour recipe meant to be in here?

**Comment by ivanholmes** [comment]
> Is the nanosour recipe meant to be in here?

No definitely not, not sure how it crept in. Do you know how I can remove it?

**Comment by Begasus** [comment]
> Do you know how I can remove it?

Not sure, think you created this branch without updating your master to upstream so no changes were left behind?

File: kde-apps/okteta/okteta-0.26.7.recipe (hunk 2 above)
Comment (ivanholmes) [reply kde-apps/okteta/okteta-0.26.7.recipe:175]: Didn't work unfortunately

File: kde-apps/okteta/okteta-0.26.7.recipe [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+	# disable docs generation
+	sed -e '/add_subdirectory( doc )/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
+
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$binDir \
```
Comment (korli) [inline kde-apps/okteta/okteta-0.26.7.recipe:170]: this one is weird

Comment (ivanholmes) [reply kde-apps/okteta/okteta-0.26.7.recipe:170]: Yes it is a bit, was based on the Okular recipe. Will investigate...

Comment (ivanholmes) [reply kde-apps/okteta/okteta-0.26.7.recipe:170]: Sorted out by @threedeyes - thanks!

**Pull request merged**